feature-libs/product/configurators/common/styles/mixins/_configurator-required-error-msg.scss

```diff
@@ -0,0 +1,10 @@
+@mixin cx-required-error-msg {
```

Comment:
This mixin is quite different from the rest of the mixins in this PR. 
- it provides style for the inner css class (instead of itself).
- the mixin name is not `-configurator-` specific
- the mixin name is not aligned with the file name

Was it on purpose?